src/Helper/PluginHelper.php

```diff
@@ -25,7 +25,7 @@ public static function isGdprPluginAdminScreen()
     {
         $screen = \get_current_screen();
 
-        return ($screen && Str::contains($screen, 'towa-gdpr-plugin'));
+        return ($screen instanceof \WP_Screen && Str::contains($screen->id, 'towa-gdpr-plugin'));
```

Comment:
@SahinU88 @phpstan discovered a critical error here

Reply:
hahah i noticed it as well. But this check is way better :)

Reply:
@volkanakb can you make a hotfix for this?

Reply:
@volkanakb Your git email is not registered here on GitHub.